tests/locators/registration.spec.js

```diff
@@ -0,0 +1,115 @@
+import {test,expect} from "@playwright/test";
+
+
+
+
+test.describe("Registration successfully", ()=>{
+    test.beforeEach(async ({page})=>{
+        await page.goto("")
+    })
+
+    test("Registration positive", async ({page})=>{
+        const signUpBtn = page.locator('button', {hasText: 'Sign up'})
+        await signUpBtn.click()
+        const popup = page.locator('.modal-content')
+        const signupNameInput = popup.locator('#signupName')
+        await signupNameInput.focus()
+        signupNameInput.fill("Elena")
+        const signupLastNameInput = popup.locator('#signupLastName')
+        await signupLastNameInput.focus()
+        signupLastNameInput.fill("Bystrova")
+        const signupEmailInput = popup.locator('#signupEmail')
+        await signupEmailInput.focus()
+        signupEmailInput.fill("aqa-elenatest@test.com")
+        const signupPasswordInput = popup.locator('#signupPassword')
+        await signupPasswordInput.focus()
+        signupPasswordInput.fill("Password123")
+        const signupRepeatPasswordInput = popup.locator('#signupRepeatPassword')
+        await signupRepeatPasswordInput.focus()
+        signupRepeatPasswordInput.fill("Password123")
+        const registerBtn = page.locator('button', {hasText: 'Register'})
+        await registerBtn.click()
+        const myProfileButton = page.locator('#userNavDropdown')
+        await expect(myProfileButton).toBeVisible()
+
+    })
+
+
+})
+
+    test.describe("Registration negative", ()=>{
+        test.beforeEach(async ({page})=>{
+            await page.goto("")
+        })
+
+        test("Empty name field", async ({page})=>{
+            const signUpBtn = page.locator('button', {hasText: 'Sign up'})
+            await signUpBtn.click()
+            const popup = page.locator('.modal-content')
+            const signupNameInput = popup.locator('#signupName')
+            await signupNameInput.focus()
+            const signupLastNameInput = popup.locator('#signupLastName')
+            await signupLastNameInput.focus()
+            signupLastNameInput.fill("Bystrova")
+            const signupEmailInput = popup.locator('#signupEmail')
+            await signupEmailInput.focus()
+            signupEmailInput.fill("aqa-elenatest@test.com")
+            const signupPasswordInput = popup.locator('#signupPassword')
+            await signupPasswordInput.focus()
+            signupPasswordInput.fill("Password123")
+            const signupRepeatPasswordInput = popup.locator('#signupRepeatPassword')
+            await signupRepeatPasswordInput.focus()
+            signupRepeatPasswordInput.fill("Password123")
+            const emptyEmailValidationText = popup.locator('p', {hasText: 'Name required'})
```

Comment:
1. make locator more precise 
2. do not rely on text, rely on selectors such as classes and check text in assertions